lib/controllers/index.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+var controllers = require.main.require('./src/controllers');
+
+var embedControllers = require('./embed');
+var adminControllers = require('./admin');
+
+controllers.embed = embedControllers;
```

Comment:
How are you making sure that these names don't conflict with those already defined by NodeBB?

Reply:
Okay, got it. Thanks.